openapi.yaml

```diff
@@ -403,30 +409,105 @@ definitions:
                 example: -123.269906
           amenities:
             type: array
+            enum: *amenitiesEnum
             items:
               type: string
-            example: [play area, bike paths]
             description: list of amenities at park
-      relationships:
+      relationships: &parkRelationships
         type: object
+        required: [owner]
         properties:
-          data:
+          owner:
             type: object
+            required: [data]
             properties:
-              type:
-                type: string
-                example: owner
-              ownerId:
-                type: string
-                example: '1111'
+              data:
+                type: object
+                required: [type, id]
+                properties:
+                  type:
+                    type: string
+                    enum: [owner]
+                  id:
+                    type: string
+                    example: '1111'
   PostParkModel:
-    required: [type, attributes]
+    required: [type, attributes, relationships]
+    properties:
+      type: *parkType
+      attributes: *parkAttributes
+      relationships: *parkRelationships
+  PatchParkModel:
+    required: [id, type]
     properties:
+      id:
+        type: string
+        example: '0001'
+        description: A unique ID of park
       type:
         type: string
         description: type of object, should be park
-        example: park
-      attributes: *parkAttributes
+        enum: [park]
+      attributes:
+        properties:
+          name:
+            type: string
+            description: Name of Park
+            example: Avery Park
+          location:
+            type: object
+            properties:
+              streetAddress:
+                type: string
+                description: Park address
+                example: '1200 SW Avery Park Dr.'
+              city:
+                type: string
+                description: City where the park is located
+                example: Corvallis
+              state:
+                type: string
+                description: Abbreviation of state where park is located
+                pattern: '^(?:(A[KLRZ]|C[AOT]|D[CE]|FL|GA|HI|I[ADLN]|K[SY]|LA|M[ADEINOST]|N[CDEHJMVY]|O[HKR]|P[AR]|RI|S[CD]|T[NX]|UT|V[AIT]|W[AIVY]))$'
+                example: OR
+              zip:
+                type: integer
+                description: Zip code where park is located
+                example: 97333
+              latitude:
+                type: number
+                format: float
+                description: latitude of park
+                example: 44.552283
+              longitude:
+                type: number
+                format: float
+                description: longitude of park
+                example: -123.269906
+          amenities:
+            type: array
+            items:
+              type: string
+            example: [play area, bike paths]
```

Comment:
Because these examples can be parsed correctly with the defined type, quotes aren't needed per the OpenAPI style guide: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#quoting-conventions